source/material_model/rheology/strain_dependent.cc

```diff
@@ -88,7 +88,12 @@ namespace aspect
                            "but is there to make sure that the original parameters for specifying the "
                            "strain weakening mechanism (``Use plastic/viscous strain weakening'') are still allowed, "
                            "but to guarantee that one uses either the old parameter names or the new ones, "
-                           "never both.");
+                           "never both."
+                           "\n\n"
+                           "If a compositional field named noninitial_plastic strain is "
```

Comment:
you need to escape this underscore that is why the documentation does not build at the moment

Reply:
Sorry, should have been more precise, double backslash like this: `\\_`

Reply:
Just saw the failed test and realized I needed a double backslash as well. Sorry about that.